server/base/src/main/java/org/apache/accumulo/server/manager/LiveTServerSet.java

```diff
@@ -228,15 +230,13 @@ public synchronized void scanServers() {
       final Set<TServerInstance> updates = new HashSet<>();
       final Set<TServerInstance> doomed = new HashSet<>();
 
-      final String path = context.getZooKeeperRoot() + Constants.ZTSERVERS;
+      Set<ServiceLockPath> tservers =
+          context.getServerPaths().getTabletServer(Optional.empty(), Optional.empty());
```

Comment:
Will this only return tservers that have a lock entry under them?  If so then that may impact the functionality in this class that cleans up tserver w/ no lock.

Reply:
Yes, it will. That's a good catch.

Reply:
Updated in 69f40d6.